src/osd/osd_types.h

```diff
@@ -4420,15 +4420,47 @@ static inline ostream& operator<<(ostream& out, const notify_info_t& n) {
 	     << " " << n.timeout << "s)";
 }
 
+struct chunk_info_t {
+  enum {
+    FLAG_DIRTY = 1, 
+    FLAG_MISSING = 2,
+  };
+  uint64_t length;
+  hobject_t oid;
+  uint64_t flags;   // FLAG_*
```

Comment:
i think we can get away with 32 bits for both these

Reply:
Right. Fixed